tests/benchmarks/scenarios/test_task_aware.py

```diff
@@ -9,19 +9,26 @@
 from avalanche.benchmarks.utils.classification_dataset import ClassificationDataset
 from avalanche.benchmarks.utils.data_attribute import DataAttribute
 from torch.utils.data import TensorDataset
+import numpy as np
 
 
 class TestsTaskAware(unittest.TestCase):
     def test_taskaware(self):
         """Common use case: add tas labels to class-incremental benchmark."""
         n_classes, n_samples_per_class, n_features = 10, 3, 7
-        dataset = make_classification(
-            n_samples=n_classes * n_samples_per_class,
-            n_classes=n_classes,
-            n_features=n_features,
-            n_informative=6,
-            n_redundant=0,
-        )
+        while True:
+            dataset = make_classification(
+                n_samples=n_classes * n_samples_per_class,
+                n_classes=n_classes,
+                n_features=n_features,
+                n_informative=6,
+                n_redundant=0,
+            )
+
+            _, unique_count = np.unique(dataset[1], return_counts=True)
+            if np.min(unique_count) > 1:
```

Comment:
What's going on here? How is this condition not always true? I'm afraid this issue may break other tests too.

Reply:
It's a corner case in which only 1 exemplar is created for one of the classes. In that case, the train test split function fails.

Reply:
ok, then it's fine. Can you add a comment about it? I hope this is not causing issues somewhere else...